.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  evaluator_job:
+    runs-on: ubuntu-18.04
+    name: Evaluator job
+    steps:
+      - name: Wait for build to succeed
+        uses: betrybe/action-wait-for-check@v2
+        id: wait-for-build
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: codeclimate
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+      - uses: actions/checkout@v2
+        if: steps.wait-for-build.outputs.conclusion == 'success'
+      - name: Evaluator step
+        id: test
+        uses: betrybe/react-scripts-evaluator-action@v3
+        if: steps.wait-for-build.outputs.conclusion == 'success'
+        with:
+          repository-test-name: 'betrybe/sd-0x-challenge-bug-hunting-youtrybe-tests'
```

Comment:
This should be `/ tryber/...`, right?

Reply:
Forest admin does that when the repo is created on **tryber** organization!